src/routes/postSkipSegments.js

```diff
@@ -101,22 +101,50 @@ async function autoModerateSubmission(submission, callback) {
         } else {
             // Check to see if video exists
             if (data.pageInfo.totalResults === 0) {
-                return "No video exists with id " + submission.videoID;
+                return "No video exists with id " + videoID;
             } else {
                 let duration = data.items[0].contentDetails.duration;
                 duration = isoDurations.toSeconds(isoDurations.parse(duration));
                 if (duration == 0) {
                     // Allow submission if the duration is 0 (bug in youtube api)
                     return false;
-                } else if ((submission.endTime - submission.startTime) > (duration/100)*80) {
-                    // Reject submission if over 80% of the video
-                    return "Sponsor segment is over 80% of the video.";
-                } else {
+                }
+
+                for (const segment of segments) {
+                    if ((segment.segment[1] - segment.segment[0]) > (duration / 100) * 80) {
+                        // Reject submission if over 80% of the video
+                        return "One of your submitted segments is over 80% of the video.";
+                    }
+                }
+
+                let neuralBlockURL = config.neuralBlockURL || "https://ai.neuralblock.app";
```

Comment:
no default address in the code - local instances and tests shouldn't be dependent on an external server